src/functionaltests/java/com/ericsson/gerrit/plugins/eiffel/linking/FunctionalTestException.java

```diff
@@ -0,0 +1,10 @@
+package com.ericsson.gerrit.plugins.eiffel.linking;
```

Comment:
copyright headers are missed, so should be added in all the files.

Reply:
I have added one header in b7112e0 but non of the other testfiles contain them so I didn't add them